csvcubed/shacl/v1/0/ttl/shapes.ttl

```diff
@@ -0,0 +1,205 @@
+<https://purl.org/csv-cubed/shacl/v1.0/deps/shacl-context.json>
```

Comment:
I assume this is procedurally generated from the easier to read JSON?

Reply:
Yes, using the bash file in the same directory.